src/settings.rs

```diff
@@ -17,8 +18,22 @@ pub struct DnsSettings {
 
 impl DnsSettings {
   pub fn load(filename: String) -> Result<Self, Box<dyn Error>> {
-    let contents = fs::read_to_string(shellexpand::full(filename.as_str()).unwrap().to_string())
-      .expect("Aw man, there was an issue while opening the config file :(");
+
+    let filenames= [filename.as_str(), "./dns.config.yaml", "~/.config/simpledns/dns.config.yaml", "/etc/simpledns/dns.config.yaml"];
+    let mut matched = false;
+    let mut contents = String::new();
+    let mut config_file = String::new();
+    for filename in filenames {    
+      contents = match fs::read_to_string(shellexpand::full(filename).unwrap().to_string()) {
```

Comment:
Can you find the config file to use without reading the whole file? 

`fs::read_from_string` returns an err result on more than if the file doesnt exist

Reply:
Done in 087f419a0cd5fa6c6ec99a0155a51475cc5178fe